README.md

```diff
@@ -1,105 +1,62 @@
 # Terraform Provider for Kentik Cloud Export
 
-## Release process
+## Requirements
 
-Release process for the provider is based on GitHub repository tags. Every tag with format v[0-9].[0-9].[0-9] will trigger automatic build of package and publish it in registry.terraform.io.
+- [Go](https://golang.org/doc/install) >= 1.17
+- [Terraform](https://www.terraform.io/downloads.html) >= 0.15
 
-To build and release package:
-1. Make sure that all code that you want to release is in master branch
-1. Create tag with format v[0-9].[0-9].[0-9] in GitHub. [Releases](https://github.com/kentik/terraform-provider-kentik-cloudexport/releases) -> Draft a new release -> Put tag version, name and description
-1. Go to [GitHub Actions](https://github.com/kentik/terraform-provider-kentik-cloudexport/actions)
+## Usage
 
-## Requirements
+Detailed user documentation for the provider can be found [here](hhttps://registry.terraform.io/providers/kentik/kentik-cloudexport/latest/docs).
```

Comment:
invalid hyperlink: hhttps

Reply:
Done

Reply:
I don't quite see the fix. There is still `hhtps://registry.terraform.io` in the URL above

Reply:
Try this:
`Detailed user documentation for the provider can be found [here](https://registry.terraform.io/providers/kentik/kentik-cloudexport/latest/docs).`

Reply:
Link fixed here too. Somehow I fixed it in Synthetics, but missed here.